app/src/androidTest/java/org/mozilla/reference/browser/ui/robots/NavigationToolbarRobot.kt

```diff
@@ -46,9 +48,11 @@ class NavigationToolbarRobot {
         }
 
         fun openThreeDotMenu(interact: ThreeDotMenuRobot.() -> Unit): ThreeDotMenuRobot.Transition {
-            mDevice.waitAndInteract(Until.findObject(By.desc("Menu"))) {
-                click()
-            }
+            mDevice.findObject(UiSelector()
+                .resourceId("org.mozilla.reference.browser.debug:id/mozac_browser_toolbar_menu"))
```

Comment:
I would write or import the same object member from Fenix that uses ```$packageName``` e.g, https://github.com/mozilla-mobile/fenix/blob/master/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt#L65

https://github.com/mozilla-mobile/fenix/blob/master/app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt#L39

Reply:
I would rather do it in a separate PR and use it where needed in the robots.

Reply:
That works.